src/CommunityToolkit/Xamarin.CommunityToolkit/PlatformConfiguration/AndroidSpecific/NavigationBar.android.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Android.OS;
+using Android.Views;
+using Xamarin.CommunityToolkit.PlatformConfiguration.Multiplatform;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.Android;
+
+namespace Xamarin.CommunityToolkit.PlatformConfiguration.AndroidSpecific
+{
+	public static partial class NavigationBar
+	{
+		static partial void SetColor(Color color)
+		{
+			if (Build.VERSION.SdkInt < BuildVersionCodes.M)
+				return;
```

Comment:
How about adding a log message that this feature isn't supported in older versions? I believe that will improve the dev xp.